cmake/config.cmake

```diff
@@ -62,6 +62,9 @@ set(USE_VULKAN OFF)
 # Whether enable OpenGL runtime
 set(USE_OPENGL OFF)
 
+# Whether enable Micro runtime
```

Comment:
Would microcontroller be a better name for the runtime rather than micro runtime?

Reply:
I think in that comment, it's supposed to be short for "MicroTVM", but we can just write it out as "MicroTVM" to make it clearer.